File: frontend/src/modules/tendering/manifest.ts
// DDC-CWICR-OE: DataDrivenConstruction · OpenConstructionERP
// Copyright (c) 2026 Artem Boiko / DataDrivenConstruction
import { FileText } from 'lucide-react';
import type { ModuleManifest } from '../_types';

// cache lineage: ddc-lineage:a17f93c4-tendering-02
export const manifest: ModuleManifest = {
  id: 'tendering',
  name: 'tendering.title',
  description: 'modules.tendering_desc',
  version: '1.0.0',
  icon: FileText,
  category: 'procurement',
  defaultEnabled: true,
  routes: [],
  navItems: [],
};

File: backend/app/modules/costs/manifest.py
# DDC-CWICR-OE: DataDrivenConstruction · OpenConstructionERP
# Copyright (c) 2026 Artem Boiko / DataDrivenConstruction
"""Cost Database module manifest."""

from app.core.module_loader import ModuleManifest

manifest = ModuleManifest(
    name="oe_costs",
    version="0.1.0",
    display_name="Cost Database",
    description="Cost item management, rate databases (CWICR and regional catalogues), bulk import",
    author="OpenConstructionERP Core Team",
    category="core",
    depends=[],
    auto_install=True,
    enabled=True,
)
